unit-tests/jvm/src/test/scala/utils/Platform.scala

```diff
@@ -0,0 +1,26 @@
+package org.scalanative.testsuite.utils
+
+// Ported from Scala.js
+
+object Platform {
+
+  final val executingInJVM = true
+
+  final val executingInScalaJS = false
```

Comment:
It is used `ArraysTest.scala`, so I imagine source compliance was the reason. An alternative would be to remove the checks in ArraysTest, but I don't really have any strong opinions about this. Perhaps it will be slightly easier in the long run if we leave it as it is.

Reply:
Let's keep it then